docs/config.md

```diff
@@ -116,6 +116,8 @@ This is a list of available options:
 | `generate.symbols`              | `bool`   | Include symbols in generated password.                                                                                                                                                                                             | `false`                             |
 | `mounts.path`                   | `string` | Path to the root store.                                                                                                                                                                                                            | `$XDG_DATA_HOME/gopass/stores/root` |
 | `notify.disable-icon`           | `bool`   | Do not show notification icon (not available on every platform). |
+| `otp.autoclip`                  | `bool`   | Alsoclip in `gopass otp` by default.                                                                                                                                                                                               | `false`                             |
+| `otp.onlyclip`                  | `bool`   | Clip in `gopass show` by default. This takes precedence on `otp.autoclip`.                                                                                                                                                         | `false`                             |
```

Comment:
The description are not very clear, I don't understand the effects of these options by just reading the descriptions right now.

Maybe:
 ```suggestion
| `otp.autoclip`                  | `bool`   | Automatically clip in `gopass otp` by default, while still displaying the codes and timers.                                                                              | `false`                             |
| `otp.onlyclip`                  | `bool`   | Automatically clip in `gopass otp` by default, without displaying the OTP codes. This takes precedence over `otp.autoclip`. Requires using `gopass otp --clip=false` to force display the codes.                                                                                       | `false`                             |
```